src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +40,36 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo?.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Why you don't remove eslint disable here? As in previous places

Reply:
I agree :)